tests/job_tests/recognition/test_scoring.py

```diff
@@ -0,0 +1,91 @@
+import os
+import tempfile
+from typing import Optional
+
+from sisyphus import tk, setup_path
+
+from i6_core.tools.compile import MakeJob
+from i6_core.tools.git import CloneGitRepositoryJob
+from i6_core.recognition import ScliteJob
+
+Path = setup_path(__package__)
+
+
+def compile_sctk(
+    branch: Optional[str] = None,
+    commit: Optional[str] = None,
+    sctk_git_repository: str = "https://github.com/usnistgov/SCTK.git",
+) -> tk.Path:
+    """
+    :param branch: specify a specific branch
+    :param commit: specify a specific commit
+    :param sctk_git_repository: where to clone SCTK from, usually does not need to be altered
+    :return: SCTK binary folder
+    """
+    sctk_job = CloneGitRepositoryJob(url=sctk_git_repository, branch=branch, commit=commit)
+    sctk_job.run()
+
+    sctk_make = MakeJob(
+        folder=sctk_job.out_repository,
+        make_sequence=["config", "all", "check", "install", "doc"],
+        link_outputs={"bin": "bin/"},
+    )
+    sctk_make.run()
+    # This is needed for the compilation to work in the i6 environment, otherwise still untested
+    sctk_make._sis_environment.set("CPPFLAGS", "-std=c++11")
+    return sctk_make.out_links["bin"]
+
+
+def test_sclite_job():
+    with tempfile.TemporaryDirectory() as tmpdir:
+        from sisyphus import gs
+
+        gs.WORK_DIR = tmpdir
+        sctk_binary = compile_sctk(branch="v2.4.12")
+        hyp = Path("files/hyp.ctm")
+        ref = Path("files/ref.stm")

```

Comment:
Do those relative paths work? Does `pytest` always changes the current dir to the module?

I would make that more explicit. You can do sth like:
```python
_my_dir = os.path.dirname(os.path.absname(__file__))
```
And then here:
```suggestion
        hyp = Path(f"{_my_dir}/files/hyp.ctm")
        ref = Path(f"{_my_dir}/files/ref.stm")
```

Reply:
Right now the test works with either the current or your variant. I dont really care which one we use.

Reply:
Why does this work without the explicit directory? Does pytest always changes the current dir to the module?

Reply:
Shouldn't `__package__` always be the location of the file? And then via `setup_path` in sisyphus it should resolve to the right path

Reply:
Actually even though both variants work, I think the one with just setup_path seems cleaner to me. But no strong opinion on this.

Reply:
Oh, you do `Path = setup_path(__package__)`. I missed that. I thought you just have `from sisyphus.job_path import Path`.

I actually thought that `setup_path` is not recommended anymore?

Reply:
I don‘t know to be honest, copied from other tests. Maybe its just deprecated in favor of gs.setup_path?
@JackTemaki @michelwi ?

Reply:
I don't know about deprecating this. If yes, I'd be interested to hear why.

Reply:
Maybe I remember wrongly.

In any case, `Path = setup_path(__package__)` is bad, it should be `rel_path = setup_path(__package__)` or so. Names with uppercase starting letter are reserved for modules. Also, you are reusing the same name of an existing class (the `Path` class), which lead to my confusion the first place.

Reply:
@michelwi https://github.com/rwth-i6/sisyphus/blob/454943ac0de00f18f4333b73f6a4002c5b742a9f/sisyphus/__init__.py#L7

Reply:
seems like just the import without tk

Reply:
Because its closer to the other tests. Why this function is in sisyphus I dont know, but since we have it I would prefer to stay consistent with the other tests.

Reply:
Well, maybe here is not the right place to reconsider this now, but in general, we don't have to repeat bad code over and over just to stay consistent with bad code.

Reply:
> setup_path and gs will be removed in the future since they are both accessible via tk

Not if I have a say about this. This will break so many of our old setups 🙈